### PR TITLE
BREAKING CHANGE: Remove feature class for bordered Alert #DS-1230

### PR DESCRIPTION
**Pull request by curdaj**

### CODE REVIEW

File: packages/web/src/scss/components/Alert/_Alert.scss [hunk 1]
```diff
@@ -13,18 +13,6 @@
     width: 100%;
     padding: theme.$padding;
     border-radius: theme.$border-radius;
-
-    // @deprecated The feature flag will be removed in the next major version.
-    // Migration: delete this feature flag and make the bordered version the default.
-    @if feature-flags.$alert-enable-bordered {
-        border: theme.$border-width theme.$border-style var(--alert-color);
```
Comment (crishpeen) [inline packages/web/src/scss/components/Alert/_Alert.scss:20]: > make the bordered version the default.

please do this too :)

**Review by crishpeen — COMMENTED** [review]
Please also update the visual tests (when everything is running, start `make test-e2e-update` and it should update the screenshot for alert). 

And remove the related section from the deprecation-v2.md file in the packages/web.

**Review by crishpeen — APPROVED** [review]
Good job! :) 💪

**Review by pavelklibani — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]
👍🏻

**Pull request merged**